### DISCUSSION TITLE
Native Microsoft Store(UWP) version of osu!lazer

### DISCUSSION THREAD

**Discussion by itsskyballs** [ppy/osu#29442] [Ideas] [answered]
osu!lazer on UWP is kinda a good idea for users who want a basic installation instead of going to the web to download it

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/23600. Will reopen that for visibility and possible upvotes.

**Reply by itsskyballs** [reply]
Thank you
